tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,246 @@
+
+
+
+.. note::
+	Godots build in input system has many things in common with the XR action map system. In fact our original idea was to add functionality to the existing input system and expose the data to the OpenXR action map system. We may revisit that idea at some point but as it turns out there were just too many problems to overcome. To name a few:
+
+	 * Godots input system mainly centers around button inputs, XR adds triggers, axis, poses and haptics (output) into the mix. This would greatly complicate the input system with features that won't work for normal controllers or contrast with the current approach. It was felt this would lead to confusion for the majority of Godot users.
+	 * Godots input system works with raw input data that is parsed and triggers emitting actions, this input data is made available to the end user. OpenXR completely hides raw data and does all the parsing for us, we only get access to already parsed action data. This inconsistency is likely to lead to bugs when an unsuspecting user tries to use an XR device as a normal input device.
+	 * Godots input system allows changes to what inputs are bound to actions in runtime, OpenXR does not.
+   * Godots input system is based on device ids which are meaningless in OpenXR.
+
+	This does mean that a game that mixes traditional inputs with XR controllers will have a separation. For most applications either one or the other is used and this is not seen as a problem. In the end, it's a limitation of the system.
+
+The default action map
+======================
+
+Godot will automatically create a default action map if no action map file is found.
+
+.. warning::
+	This default map was designed to help developers port their XR games from Godot 3 to Godot 4. As a result this map essentially binds all known inputs on all controllers supported by default, to actions one on one. This is not a good example of setting up an action map but it does allow a new developer to have a starting point when they want to become familiar with Godot XR without designing a proper action map for their game first.
+
+For this walk through we're going to start with a blank action map. You can simply delete the "Godot action set" entry at the top by pressing the trash can icon and it will clear out all actions. You can also remove the controllers that you do not wish to setup, more on this later.
+
+Action sets
+===========
+
+.. note::
+  Before we dive in, you will see the term XR runtime used throughout this document. With XR runtime we mean the software that is controlling and interacting with the AR or VR headset and exposed to us through an API such as OpenXR. So for Steam this is SteamVR, for Meta on desktop this is the Oculus Client (including when using Quest link), for Meta on Quest this is the Quests native OpenXR client, on Linux this could be Monado, etc.
+
+The action map allows us to organise our actions in sets. These sets can be individually enabled/disabled.
```

Comment:
No, I always make those mistakes so thanks for pointing them out. I don't know if it is because I live in Oz or whether I learned English in Holland but I just can't seem to shake defaulting to British spelling.